python/docs/source/usage.rst

```diff
@@ -70,8 +70,9 @@ For AWS Glue catalog, use AWS environment variables to authenticate.
     >>> dt.to_pyarrow_table().to_pydict()
     {'id': [5, 7, 9, 5, 6, 7, 8, 9]}
 
-.. _`s3 options`: https://github.com/delta-io/delta-rs/blob/17999d24a58fb4c98c6280b9e57842c346b4603a/rust/src/builder.rs#L423-L491
-.. _`azure options`: https://github.com/delta-io/delta-rs/blob/17999d24a58fb4c98c6280b9e57842c346b4603a/rust/src/builder.rs#L524-L539
+.. _`s3 options`: https://docs.rs/object_store/0.5.3/object_store/aws/enum.AmazonS3ConfigKey.html#variants
+.. _`azure options`: https://docs.rs/object_store/0.5.3/object_store/azure/enum.AzureConfigKey.html#variants
+.. _`gcs options`: https://docs.rs/object_store/0.5.3/object_store/gcp/enum.GoogleConfigKey.html#variants
```

Comment:
We probably won't remember to update this, so maybe the best thing is to point to the latest version?

```suggestion
.. _`s3 options`: https://docs.rs/object_store/latest/object_store/aws/enum.AmazonS3ConfigKey.html#variants
.. _`azure options`: https://docs.rs/object_store/latest/object_store/azure/enum.AzureConfigKey.html#variants
.. _`gcs options`: https://docs.rs/object_store/latest/object_store/gcp/enum.GoogleConfigKey.html#variants
```